docs/installation.md

```diff
@@ -1,15 +1,15 @@
 ## Using Docker
 
 ```
-docker pull brownbnc/xnat_tools:<version>
+docker pull brownbnc/xnat-tools:<version>
 ```
 
 _Version:_
 
 - `latest`: Is the build of main
 - `vX.X.X`: Latest tagged stable release
 
-You can confirm the tags [here](https://hub.docker.com/repository/docker/brownbnc/xnat_tools/tags?page=1)
+You can confirm the tags [here](https://hub.docker.com/repository/docker/brownbnc/xnat-tools/tags?page=1)
```

Comment:
looks like this should just be https://hub.docker.com/r/brownbnc/xnat-tools/tags